WINDOWS-TEST.md

```diff
@@ -0,0 +1,40 @@
+# Envoy Integration Tests on Windows
```

Comment:
can you move this file from the root to `test/integration/connect/envoy/WINDOWS-TEST.md`